CMakeLists.txt

```diff
@@ -31,7 +32,11 @@ list(APPEND SOURCE ${PROJECT_SOURCE_DIR}/src/utils.cpp
 include_directories(src)
 include_directories(include)
 
-add_subdirectory(test)
+if(ENABLE_TESTS)
+    include(CTest)

```

Comment:
Should this line go into the `tests/CMakeLists.txt` file instead?

Reply:
I tried this also, but then the ctest command is only available in the test folder it self, so in the CI it would be then ``cd build test` and then `ctest`. I am not sure what is the correct way to do.

Reply:
According to this document: https://cmake.org/cmake/help/latest/module/CTest.html
the CTest shall be in the top CMakeLists.txt

Reply:
Alright that is good know. Thank you for tracking that down!